uk.co.compucorp.civicrm.tasksassignments/js/test/tasks-assignments/controllers/document-list.controller.spec.js

```diff
@@ -382,7 +480,35 @@ define([
       });
     });
 
-    function initController (scopeValues) {
+    /**
+     * Fabricates document statuses and sets them to the root scope cache
+     */
+    function fabricateDocumentStatuses () {
+      _.each(documentFabricator.documentStatus(), function (option) {
+        $rootScope.cache.documentStatus.obj[option.key] = option.value;
+        $rootScope.cache.documentStatus.arr.push(option);
+      });
+    }
+
+    /**
+     * Gets document statuses IDs by their names
+     *
+     * @param  {Array} statusesNames
+     * @return {Array} statuses IDs
```

Comment:
is a space between @param and @return required? I've been working with the php linter so much I don't remember too well how it is for JS code anymore :(

Reply:
Oh, you must have worked with php a lot! In JS we don't use a space, I am 99% sure this is a correct and consistent syntax we use everywhere in JS.